lib/modules/saveComments.js

```diff
@@ -54,6 +76,117 @@ module.go = async () => {
 	}
 };
 
+// Comment save on reply implementation
+
+function addReplyListener(thing) {
+	if (thing) {
+		const replyButton = thing.element.querySelector('a.access-required[data-event-action=comment]');
```

Comment:
As @andytuba mentioned, it might be more reliable to listen to new comments and check whether the user is the owner, like is done here: https://github.com/honestbleeps/Reddit-Enhancement-Suite/blob/b287af15f1a7197f6d29f3acefeea776e7fcde21/lib/modules/newCommentCount.js#L183-L186